src/averageAges.js

```diff
@@ -20,6 +20,12 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const menAgeArr = (people.filter(person => century
+    ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+    : person.sex === 'm'))
+    .map(person => person.died - person.born);
+
+  return menAgeArr.reduce((a, b) => a + b, 0) / menAgeArr.length;
```

Comment:
Don't repeat yourself
write callback for this calculations and use it on 28, 51 and 76 lines)